src/client/gui/lib/update_available.dart

```diff
@@ -15,14 +17,39 @@ class UpdateNotifier extends Notifier<UpdateInfo> {
 
   void set(UpdateInfo updateInfo) {
     if (updateInfo.version.isBlank) return;
+
     final updateNotificationExists = ref.read(notificationsProvider).any((n) {
       return n is UpdateAvailableNotification && n.updateInfo == updateInfo;
     });
     if (updateNotificationExists) return;
-    ref
-        .read(notificationsProvider.notifier)
-        .add(UpdateAvailableNotification(updateInfo));
+
+    // Existing in-app notification
```

Comment:
I think just `In-app notification` would be enough for this comment. Specifying that it's `existing` does not clarify much.